src/components/LibrariesPage.tsx

```diff
@@ -99,7 +99,8 @@ export class LibrariesPage extends React.Component<LibrariesPageProps, Libraries
   }
 
   filter(libraries) {
```

Comment:
Not necessary now but it'll be good to update this to be `filterProduction` so it's obvious from the function name that it's filtering for libraries in production, since that's the only filtering that's happening.

Reply:
Good call; will go ahead and do that now.